miraclelinux/meta-emlinux
# Revert "Merge pull request #139 from masami256/import-pseudo-fix-patch"

# Purpose of pull request

meta-debian's pull request #297 has been merged[1]. Therefore it's time
to remove pesudo fix from this repo.

1: https://github.com/meta-debian/meta-debian/pull/297

# Test
## How to test

1. check pseudo package doesn't exit in meta-emlinux
2. build core-image-weston without any warnings

## Test result

recipes.

```
masami@ubuntu1804:~/emlinux/qemuarm64-emlinux$ bitbake-layers show-recipes | grep -A 4 pseudo
pseudo:
  meta-debian          1.9.0+git20180920
  meta                 1.9.0+gitAUTOINC+060058bb29
psmisc:
  meta-debian-extended 23.2
```

build result(I added -f option so that " forced run" warnings are displayed. but it can be ignored.)

```
masami@ubuntu1804:~/emlinux/qemuarm64-emlinux$ bitbake -f core-image-weston
Loading cache: 100% |##########################################################################################################################################################################################################| Time: 0:00:00
Loaded 2359 entries from dependency cache.
NOTE: Resolving any missing task queue dependencies

Build Configuration:
BB_VERSION           = "1.42.0"
BUILD_SYS            = "x86_64-linux"
NATIVELSBSTRING      = "ubuntu-18.04"
TARGET_SYS           = "aarch64-emlinux-linux"
MACHINE              = "qemuarm64"
DISTRO               = "emlinux"
DISTRO_VERSION       = "2.4"
TUNE_FEATURES        = "aarch64 armv8a crc"
TARGET_FPU           = ""
meta                 
meta-yocto-bsp       = "warrior:d4b57c68b22027c2bedff335dee06af963e4f8a8"
meta-debian          = "warrior:7efa6b4f08da8d88e413d7bc482ac85825e8d0c8"
meta-debian-extended = "warrior:ee8c1e7caaf224db9f3881a8ef874eab8c35914e"
meta-emlinux         = "revert-pseudo-patch:581bb6a3e4b8052f8b3171b48d8ef75448a1cf38"

NOTE: Tainting hash to force rebuild of task /home/masami/emlinux/qemuarm64-emlinux/../repos/poky/meta/recipes-graphics/images/core-image-weston.bb, do_build                                                                  | ETA:  0:00:00
WARNING: /home/masami/emlinux/qemuarm64-emlinux/../repos/meta-debian-extended/recipes-debian/coreutils/coreutils_debian.bb.do_build is tainted from a forced run                                                               | ETA:  0:00:00
WARNING: /home/masami/emlinux/qemuarm64-emlinux/../repos/poky/meta/recipes-graphics/images/core-image-weston.bb.do_build is tainted from a forced run
Initialising tasks: 100% |#####################################################################################################################################################################################################| Time: 0:00:02
Sstate summary: Wanted 1135 Found 0 Missed 1135 Current 427 (0% match, 27% complete)
NOTE: Executing SetScene Tasks
NOTE: Executing RunQueue Tasks
NOTE: Tasks Summary: Attempted 5136 tasks of which 1975 didn't need to be rerun and all succeeded.

Summary: There were 2 WARNING messages shown.
```
